extensionmanager/src/main/resources/qupath/ext/extensionmanager/strings.properties

```diff
@@ -44,11 +44,13 @@ ManuallyInstalledExtensionLine.cannotDeleteExtension = Cannot delete extension:\
 ProgressWindow.cancel = Cancel
 
 Catalog.ExtensionDetails.close = Close
+Catalog.ExtensionDetails.extensionNotCompatible = This extension is not compatible with this version.
```

Comment:
Suggest 
```
This extension is not compatible with this version of QuPath.
```
or
```
This extension is not compatible with the current version of QuPath.
```
to make clear that 'this version' doesn't refer to the same software as 'this extension'.